.github/pull_request_template.md

```diff
@@ -6,3 +6,12 @@
 ## Solution
 
 - Describe the solution used to achieve the objective above.
+
+## Changelog
```

Comment:
Should this include the `### Added`, `### Changed`, etc. headers?

Reply:
This feels like a lot of boilerplate for small PRs. I would maybe only add a comment suggesting using those sections

Reply:
```markdown
- What changed as a result of this PR?
- If applicable, organize changes under "Added", "Changed", or "Fixed" sub-headings
```

Thoughts?

Reply:
LGTM